tests/portability/cmsis_rtos_v2/src/thread_apis.c

```diff
@@ -284,6 +284,7 @@ void test_thread_join(void)
 	zassert_not_null(tB, "Failed to create thread B with osThreadNew!");
 
 	printk(" - Creating thread A...\n");
+	attr.priority = osPriorityLow;
```

Comment:
+1

Reply:
can you update the commit message? it still references a delay

Reply:
Done, thanks for reviewing it